engine/apps/api/serializers/alert_group.py

```diff
@@ -204,12 +204,7 @@ def get_limited_alerts(self, obj):
         Overriding default alerts because there are alert_groups with thousands of them.
         It's just too slow, we need to cut here.
         """
-        alerts = obj.alerts.all()[:100]
-
-        if len(alerts) > 90:
-            for alert in alerts:
-                alert.title = str(alert.title) + " Only last 100 alerts are shown. Use OnCall API to fetch all of them."
```

Comment:
Will we show any message on frontend or mention in docs, that we are showing only last 100 alerts?

Reply:
The "Only last 100 alerts are shown ..." section wasn't working correctly anyways because `alert.title` is not used to render alerts. So removing this doesn't change anything, we should add a disclaimer or implement pagination in a separate PR.